examples/postgres-claim.yaml

```diff
@@ -4,6 +4,8 @@ metadata:
   name: platform-ref-aws-db-postgres
 spec:
   parameters:
+    region: us-west-2
+    deletionPolicy: Delete
```

Comment:
Same as the above

Reply:
@stevendborrelli asked for it to be included. Going forward, I think we should keep PRs to the original intent and create new ones for new requests.